### PR TITLE
Prime command

### PR DESCRIPTION
**Pull request by abidina**
what is logic? what is math?  @katerebekah       ¯_(ツ)_/¯

### CODE REVIEW

File: Mathmagician/Prime.cs [hunk 1]
```diff
@@ -18,10 +18,51 @@ public int FollowingPrime(int v)
             return 3;
         }
 
-        public List<int> Sequence(int v)
+ /*       public static bool isPrime(int increasingNumber)
```
Comment (katerebekah) [inline Mathmagician/Prime.cs:21]: Be sure to clean this up before the project is done. (I know you might need it right now)

File: Mathmagician/Prime.cs [hunk 2]
```diff
@@ -18,10 +18,51 @@ public int FollowingPrime(int v)
...
+                    {
+                        isPrime = false;
+                        break;
+                    }
+                }
+                if (isPrime)
+                {
+                    PrimeOutputList.Add(followingPrime);
+                }
+                followingPrime += 2;
```
Comment (katerebekah) [inline Mathmagician/Prime.cs:63]: Is this so you don't test even numbers? if so, cool. If not, what is it doing?

**Comment by katerebekah** [comment]
What is is? What is the meaning of life and primes?

**Comment by katerebekah** [comment]
If you the + 2 is for avoiding even numbers then :+1:

**Comment by abidina** [comment]
@katerebekah pls see line 46, comment "ONLY TESTS ODD NUMBERS" bitmoji

**Comment by katerebekah** [comment]
Love it!

**Comment by csbarrett1** [comment]
👍

**Pull request merged**